priorityq/priorityq_test.go

```diff
@@ -109,5 +109,5 @@ func TestPriorityQ_Read(t *testing.T) {
 	time.Sleep(1500 * time.Millisecond)
 	pq.Close()
 	rg.Wait()
-	r.Equal(3*defLen, i-1)
```

Comment:
Good.
Also improve the test by:
1. i++ after the if ok.
2. in the if ok don't break, do FailNow